src/content/blog/community/evaluating-prompt-injection.md

```diff
@@ -0,0 +1,683 @@
+---
+title: 'Evaluating prompt injection guardrails' 
+description: 'Here is a sample of some basic Markdown syntax that can be used when writing Markdown content in Astro.'
+pubDate: 'Oct 28 2025'
+heroImage: '/blog-placeholder-3.jpg'
+track: 'community'
+authors: ['mac']
+---
+
+## Introduction
+
+Based on the [OWASP](https://owasp.org/) Top 10 risks and vulnerabilities ranking, prompt injection has been the number one risk for large language models (LLMs) and Generative Artificial Intelligence (GenAI) applications in both [2024](https://genai.owasp.org/llmrisk2023-24/llm01-24-prompt-injection/) and [2025](https://genai.owasp.org/llmrisk/llm01-prompt-injection/). Evidently, prompt injection is a considerable cyber security threat and requires effective mitigation strategies. Among the most common mitigation strategies are content moderation guardrails (from here on we will refer to these as _guardrails_ for the sake of brevity). These  guardrails aim to filter out unwanted content at input and / or output stages of text generation. A variety of different techniques can be used to develop such guardrails. These technqiues can encompass rule-based approaches (i.e. specifying regex expressions) and model-based approaches (i.e. using classification models). Subsequently, the aforementioned guardrails are usually consumed by some kind of orchestration layer, which manages the interaction between the user, guardrails and LLM, for example [IBM FMS Orchestrator](https://github.com/foundation-model-stack/fms-guardrails-orchestrator?tab=readme-ov-file) or [Nvidia NeMo](https://github.com/NVIDIA-NeMo/Guardrails). 
+
+Finding suitable prompt injection guardrails is a difficult task in itself, as the landscape appears to be rather fragmented with many different open-source and commercial offerings. Moreover, evaluating the effectiveness of prompt injection guardrails is far from trivial and poses several challenges, for example
+
+- prompt injection attacks drift over time as attackers find novel ways to circumvent defenses;
+- there is a lack of uniformity in how prompt injection guardrails are benchmarked
+
+In this blog post, we aim to partially tackle the aforementioned challenges by attempting to systematically find candidate prompt injection guardrails and evaluate them against a common set of prompt injection attacks.
+
+## Finding candidate prompt injection guardrails
+
+We restricted the search space to guardrail models that are
+
+- open source
+- Transformer-based with a binary classifier head (i.e. `AutoModelsForSequenceClassification` models that produce scores for each of the two labels)
+- fine-tuned on prompt injection data
+- downloadable from [Hugging Face Models Hub](https://huggingface.co/models). 
+
```

Comment:
It might be interesting to note why a model like https://huggingface.co/meta-llama/Prompt-Guard-86M that may seem well known did not make it here. I'm presuming by looking at the scripts that the license did not fall into the list, but it may be good to call out via criteria?